.github/actions/update-wiki/action.yml

```diff
@@ -0,0 +1,29 @@
+# Runs update-wiki update actions every day at 20:00 UTC
+
+name: Update Wiki
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think the on section is not needed here as the update-wiki action is triggered by the wiki workflow.